src/Faker/Provider/id_ID/Company.php

```diff
@@ -15,6 +15,15 @@ class Company extends \Faker\Provider\Company
      * @see http://id.wikipedia.org/wiki/Jenis_badan_usaha
      */
     protected static $companyPrefix = ['PT', 'CV', 'UD', 'PD', 'Perum'];
+    
+
```

Comment:
Remove extra newline

Reply:
Extra newline removed

Reply:
> Extra newline removed ([fb28453](https://github.com/FakerPHP/Faker/pull/338/commits/fb28453718417e6dcf3582cfdf14b07bf3c517c3))

Build still failing. Please review the codestyle